packages/react-ui-components/src/MultiSelectBox/multiSelectBox.js

```diff
@@ -31,6 +31,11 @@ class MultiSelectBox extends PureComponent {
          */
         disabled: PropTypes.bool,
 
+        /**
+         * Allows dragging.
+         */
+        allowDragging: PropTypes.bool,
```

Comment:
im not totally found of the `allow*` naming schema, but it seems were following the pattern of `allowEmpty` so it makes sense :D

Reply:
Could also be `enableDragging` if you like that more?

Reply:
no thanks I was just reflecting